draft-smyslov-ipsecme-rfc8229bis.xml

```diff
@@ -230,27 +230,26 @@
         of a message to differentiate IKE and ESP messages.  TCP
         encapsulation also adds a 16-bit Length field that precedes every message
         to define the boundaries of messages within a stream.  
-        The message length plus the length of the field itself is sent this field.
-        If the first 32 bits of the message are zeros (a non-ESP marker), then the contents comprise an
+		The value in this field is equal to the length of the original message
+		plus the length of the field itself, in octets.  If the first 32 bits
+		of the message are zeros (a non-ESP marker), then the contents comprise an
         IKE message.  Otherwise, the contents comprise an ESP message.
         Authentication Header (AH) messages are not supported for TCP
         encapsulation.
         </t>
   
-        <t>Note, that since TCP header is longer than UDP header, and TCP
-        encapsulation also requires prepending of 16-bit Length field, some
-        very long ESP and IKE messages that could be sent over UDP cannot be
-        encapsulated in TCP, because their total length after encapsulation
-        would exceed 65535 and thus could not be represented in Length field.
-        </t>
-  
         <t>Although a TCP stream may be able to send very long messages,
         implementations SHOULD limit message lengths to typical UDP datagram
         ESP payload lengths.  The maximum message length is used as the
         effective MTU for connections that are being encrypted using ESP, so
         the maximum message length will influence characteristics of inner
         connections, such as the TCP Maximum Segment Size (MSS).
-        </t>
+        Additionally, since TCP headers are longer than UDP headers, and TCP
```

Comment:
This would either be "...since a TCP header is longer than a UDP header..." or "...since TCP headers are longer than UDP headers...". If its singular, it needs an article like "a" or "the". I opted for plural since this is true of all TCP and UDP headers, and we're referring to the fact of the matter for many packets.

Reply:
OK